Cannot build on Python 3.9

`pip install udatetime` fails to compile `udatetime` on Python 3.9. Works fine on 3.8.

Tested on:

* Python 3.9.0 on Ubuntu in Windows Subsystem for Linux on Windows 10
* Python 3.9.0 on Darwin 19.6.0 (MacBook)

Fails with similar errors on both.

----

<details>
  <summary>Stacktrace for Ubuntu in WSL</summary>

```
$ env39_wsl/bin/pip install udatetime
Collecting udatetime
  Downloading udatetime-0.0.16.tar.gz (13 kB)
Using legacy 'setup.py install' for udatetime, since package 'wheel' is not installed.
Installing collected packages: udatetime
    Running setup.py install for udatetime ... error
    ERROR: Command errored out with exit status 1:
     command: /mnt/c/Users/<redacted>/env39_wsl/bin/python3.9 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-1qm3faee/udatetime/setup.py'"'"'; __file__='"'"'/tmp/pip-install-1qm3faee/udatetime/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-lkuefh40/install-record.txt --single-version-externally-managed --compile --install-headers /mnt/c/Users/<redacted>/env39_wsl/include/site/python3.9/udatetime
         cwd: /tmp/pip-install-1qm3faee/udatetime/
    Complete output (36 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-3.9
    creating build/lib.linux-x86_64-3.9/udatetime
    copying udatetime/__init__.py -> build/lib.linux-x86_64-3.9/udatetime
    copying udatetime/_pure.py -> build/lib.linux-x86_64-3.9/udatetime
    running egg_info
    writing udatetime.egg-info/PKG-INFO
    writing dependency_links to udatetime.egg-info/dependency_links.txt
    writing top-level names to udatetime.egg-info/top_level.txt
    reading manifest file 'udatetime.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'udatetime.egg-info/SOURCES.txt'
    running build_ext
    building 'udatetime.rfc3339' extension
    creating build/temp.linux-x86_64-3.9
    creating build/temp.linux-x86_64-3.9/src
    x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security
 -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -D_PYTHON3=1 -I/mnt/c/Users/<redacted>/env39_wsl/include -I/usr/
include/python3.9 -c ./src/rfc3339.c -o build/temp.linux-x86_64-3.9/./src/rfc3339.o -Ofast -std=c99
    ./src/rfc3339.c: In function ‘_gettime’:
    ./src/rfc3339.c:100:18: error: storage size of ‘t’ isn’t known
         struct timeb t;
                      ^
    ./src/rfc3339.c:101:5: warning: implicit declaration of function ‘ftime’; did you mean ‘stime’? [-Wimplicit-function-declaration]
         ftime(&t);
         ^~~~~
         stime
    ./src/rfc3339.c:100:18: warning: unused variable ‘t’ [-Wunused-variable]
         struct timeb t;
                      ^
    ./src/rfc3339.c: At top level:
    ./src/rfc3339.c:516:21: warning: ‘CAPI’ initialized and declared ‘extern’
     extern RFC3999_CAPI CAPI = {
                         ^~~~
    error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /mnt/c/Users/<redacted>/env39_wsl/bin/python3.9 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-1qm3faee/udatetime/setup.py'"'"'; __file__='"'"'/tmp/pip-install-1qm3faee/udatetime/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-lkuefh40/install-record.txt --single-version-externally-managed --compile --install-headers /mnt/c/Users/<redacted>/env39_wsl/include/site/python3.9/udatetime Check the logs for full command output.
```
</details>


<details>
  <summary>Stacktrace for Darwin (MacBook)</summary>
```
Building wheels for collected packages: udatetime
  Building wheel for udatetime (setup.py): started
  Building wheel for udatetime (setup.py): finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /Users/michael.overmeyer/src/github.com/closeio/ciso8601/benchmarking/.tox/py39/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-install-sjrwiw8m/udatetime/setup.py'"'"'; __file__='"'"'/private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-install-sjrwiw8m/udatetime/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-wheel-ar7o0x3d
       cwd: /private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-install-sjrwiw8m/udatetime/
  Complete output (34 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.9-x86_64-3.9
  creating build/lib.macosx-10.9-x86_64-3.9/udatetime
  copying udatetime/__init__.py -> build/lib.macosx-10.9-x86_64-3.9/udatetime
  copying udatetime/_pure.py -> build/lib.macosx-10.9-x86_64-3.9/udatetime
  running egg_info
  writing udatetime.egg-info/PKG-INFO
  writing dependency_links to udatetime.egg-info/dependency_links.txt
  writing top-level names to udatetime.egg-info/top_level.txt
  reading manifest file 'udatetime.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  writing manifest file 'udatetime.egg-info/SOURCES.txt'
  running build_ext
  building 'udatetime.rfc3339' extension
  creating build/temp.macosx-10.9-x86_64-3.9
  creating build/temp.macosx-10.9-x86_64-3.9/src
  gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch x86_64 -g -D_PYTHON3=1 -I/Users/michael.overmeyer/src/github.com/closeio/ciso8601/benchmarking/.tox/py39/include -I/Library/Frameworks/Python.framework/Versions/3.9/include/python3.9 -c ./src/rfc3339.c -o build/temp.macosx-10.9-x86_64-3.9/./src/rfc3339.o -Ofast -std=c99
  ./src/rfc3339.c:100:18: error: variable has incomplete type 'struct timeb'
      struct timeb t;
                   ^
  ./src/rfc3339.c:100:12: note: forward declaration of 'struct timeb'
      struct timeb t;
             ^
  ./src/rfc3339.c:101:5: error: implicit declaration of function 'ftime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      ftime(&t);
      ^
  ./src/rfc3339.c:516:21: warning: 'extern' variable has an initializer [-Wextern-initializer]
  extern RFC3999_CAPI CAPI = {
                      ^
  1 warning and 2 errors generated.
  error: command '/usr/bin/gcc' failed with exit code 1
  ----------------------------------------
  ERROR: Failed building wheel for udatetime
  Running setup.py clean for udatetime
Failed to build udatetime
Installing collected packages: aniso8601, six, python-dateutil, arrow, iso8601, monthdelta, enum34, iso8601utils, isodate, pytz, tzlocal, regex, dateparser, humanize, snaptime, pytzdata, pendulum, maya, times, moment, pyso8601, str2date, udatetime, pytimeparse, Babel, zulu, chardet, mbstrdecoder, typepy, msgfy, tcolorpy, DataProperty, pathvalidate, tabledata, pytablewriter, toml, filelock, appdirs, distlib, virtualenv, py, pluggy, pyparsing, packaging, tox
    Running setup.py install for udatetime: started
    Running setup.py install for udatetime: finished with status 'error'
    ERROR: Command errored out with exit status 1:
     command: /Users/michael.overmeyer/src/github.com/closeio/ciso8601/benchmarking/.tox/py39/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-install-sjrwiw8m/udatetime/setup.py'"'"'; __file__='"'"'/private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-install-sjrwiw8m/udatetime/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-record-ooxk6ok5/install-record.txt --single-version-externally-managed --compile --install-headers /Users/michael.overmeyer/src/github.com/closeio/ciso8601/benchmarking/.tox/py39/include/site/python3.9/udatetime
         cwd: /private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-install-sjrwiw8m/udatetime/
    Complete output (34 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.9-x86_64-3.9
    creating build/lib.macosx-10.9-x86_64-3.9/udatetime
    copying udatetime/__init__.py -> build/lib.macosx-10.9-x86_64-3.9/udatetime
    copying udatetime/_pure.py -> build/lib.macosx-10.9-x86_64-3.9/udatetime
    running egg_info
    writing udatetime.egg-info/PKG-INFO
    writing dependency_links to udatetime.egg-info/dependency_links.txt
    writing top-level names to udatetime.egg-info/top_level.txt
    reading manifest file 'udatetime.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'udatetime.egg-info/SOURCES.txt'
    running build_ext
    building 'udatetime.rfc3339' extension
    creating build/temp.macosx-10.9-x86_64-3.9
    creating build/temp.macosx-10.9-x86_64-3.9/src
    gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch x86_64 -g -D_PYTHON3=1 -I/Users/michael.overmeyer/src/github.com/closeio/ciso8601/benchmarking/.tox/py39/include -I/Library/Frameworks/Python.framework/Versions/3.9/include/python3.9 -c ./src/rfc3339.c -o build/temp.macosx-10.9-x86_64-3.9/./src/rfc3339.o -Ofast -std=c99
    ./src/rfc3339.c:100:18: error: variable has incomplete type 'struct timeb'
        struct timeb t;
                     ^
    ./src/rfc3339.c:100:12: note: forward declaration of 'struct timeb'
        struct timeb t;
               ^
    ./src/rfc3339.c:101:5: error: implicit declaration of function 'ftime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        ftime(&t);
        ^
    ./src/rfc3339.c:516:21: warning: 'extern' variable has an initializer [-Wextern-initializer]
    extern RFC3999_CAPI CAPI = {
                        ^
    1 warning and 2 errors generated.
    error: command '/usr/bin/gcc' failed with exit code 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /Users/michael.overmeyer/src/github.com/closeio/ciso8601/benchmarking/.tox/py39/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-install-sjrwiw8m/udatetime/setup.py'"'"'; __file__='"'"'/private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-install-sjrwiw8m/udatetime/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/f5/5752mxwn72j_xmm44mc5xt900000gn/T/pip-record-ooxk6ok5/install-record.txt --single-version-externally-managed --compile --install-headers /Users/michael.overmeyer/src/github.com/closeio/ciso8601/benchmarking/.tox/py39/include/site/python3.9/udatetime Check the logs for full command output.
```
</details>

Hello, Same issue on Ubuntu. If it helps, I have attached stack trace below

OS: Ubuntu 21.04
Python: 3.9.5

<details>
  <summary>StackTrace</summary>
  


```
[pipenv.exceptions.InstallError]: Collecting udatetime==0.0.16
[pipenv.exceptions.InstallError]:   Using cached udatetime-0.0.16.tar.gz (13 kB)
[pipenv.exceptions.InstallError]:   Preparing metadata (setup.py): started
[pipenv.exceptions.InstallError]:   Preparing metadata (setup.py): finished with status 'done'
[pipenv.exceptions.InstallError]: Building wheels for collected packages: udatetime
[pipenv.exceptions.InstallError]:   Building wheel for udatetime (setup.py): started
[pipenv.exceptions.InstallError]:   Building wheel for udatetime (setup.py): finished with status 'error'
[pipenv.exceptions.InstallError]:   Running setup.py clean for udatetime
[pipenv.exceptions.InstallError]: Failed to build udatetime
[pipenv.exceptions.InstallError]: Installing collected packages: udatetime
[pipenv.exceptions.InstallError]:   Running setup.py install for udatetime: started
[pipenv.exceptions.InstallError]:   Running setup.py install for udatetime: finished with status 'error'
[pipenv.exceptions.InstallError]: error: subprocess-exited-with-error
[pipenv.exceptions.InstallError]:   
[pipenv.exceptions.InstallError]:   × python setup.py bdist_wheel did not run successfully.
[pipenv.exceptions.InstallError]:   │ exit code: 1
[pipenv.exceptions.InstallError]:   ╰─> [38 lines of output]
[pipenv.exceptions.InstallError]:       running bdist_wheel
[pipenv.exceptions.InstallError]:       running build
[pipenv.exceptions.InstallError]:       running build_py
[pipenv.exceptions.InstallError]:       creating build
[pipenv.exceptions.InstallError]:       creating build/lib.linux-x86_64-cpython-39
[pipenv.exceptions.InstallError]:       creating build/lib.linux-x86_64-cpython-39/udatetime
[pipenv.exceptions.InstallError]:       copying udatetime/_pure.py -> build/lib.linux-x86_64-cpython-39/udatetime
[pipenv.exceptions.InstallError]:       copying udatetime/__init__.py -> build/lib.linux-x86_64-cpython-39/udatetime
[pipenv.exceptions.InstallError]:       running egg_info
[pipenv.exceptions.InstallError]:       writing udatetime.egg-info/PKG-INFO
[pipenv.exceptions.InstallError]:       writing dependency_links to udatetime.egg-info/dependency_links.txt
[pipenv.exceptions.InstallError]:       writing top-level names to udatetime.egg-info/top_level.txt
[pipenv.exceptions.InstallError]:       reading manifest file 'udatetime.egg-info/SOURCES.txt'
[pipenv.exceptions.InstallError]:       reading manifest template 'MANIFEST.in'
[pipenv.exceptions.InstallError]:       writing manifest file 'udatetime.egg-info/SOURCES.txt'
[pipenv.exceptions.InstallError]:       warning: build_py: byte-compiling is disabled, skipping.
[pipenv.exceptions.InstallError]:       
[pipenv.exceptions.InstallError]:       running build_ext
[pipenv.exceptions.InstallError]:       building 'udatetime.rfc3339' extension
[pipenv.exceptions.InstallError]:       creating build/temp.linux-x86_64-cpython-39
[pipenv.exceptions.InstallError]:       creating build/temp.linux-x86_64-cpython-39/src
[pipenv.exceptions.InstallError]:       x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -ffile-prefix-map=/build/python3.9-FZ7wim/python3.9-3.9.5=. -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -D_PYTHON3=1 -I/home/twista/.local/share/virtualenvs/tasks-Ja8_01_a/include -I/usr/include/python3.9 -c ./src/rfc3339.c -o build/temp.linux-x86_64-cpython-39/./src/rfc3339.o -Ofast -std=c99
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c: In function ‘_gettime’:
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c:100:18: error: storage size of ‘t’ isn’t known
[pipenv.exceptions.InstallError]:         100 |     struct timeb t;
[pipenv.exceptions.InstallError]:             |                  ^
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c:101:5: warning: implicit declaration of function ‘ftime’; did you mean ‘ctime’? [-Wimplicit-function-declaration]
[pipenv.exceptions.InstallError]:         101 |     ftime(&t);
[pipenv.exceptions.InstallError]:             |     ^~~~~
[pipenv.exceptions.InstallError]:             |     ctime
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c:100:18: warning: unused variable ‘t’ [-Wunused-variable]
[pipenv.exceptions.InstallError]:         100 |     struct timeb t;
[pipenv.exceptions.InstallError]:             |                  ^
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c: At top level:
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c:516:21: warning: ‘CAPI’ initialized and declared ‘extern’
[pipenv.exceptions.InstallError]:         516 | extern RFC3999_CAPI CAPI = {
[pipenv.exceptions.InstallError]:             |                     ^~~~
[pipenv.exceptions.InstallError]:       error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
[pipenv.exceptions.InstallError]:       [end of output]
[pipenv.exceptions.InstallError]:   
[pipenv.exceptions.InstallError]:   note: This error originates from a subprocess, and is likely not a problem with pip.
[pipenv.exceptions.InstallError]:   ERROR: Failed building wheel for udatetime
[pipenv.exceptions.InstallError]:   error: subprocess-exited-with-error
[pipenv.exceptions.InstallError]:   
[pipenv.exceptions.InstallError]:   × Running setup.py install for udatetime did not run successfully.
[pipenv.exceptions.InstallError]:   │ exit code: 1
[pipenv.exceptions.InstallError]:   ╰─> [40 lines of output]
[pipenv.exceptions.InstallError]:       running install
[pipenv.exceptions.InstallError]:       /home/twista/.local/share/virtualenvs/tasks-Ja8_01_a/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
[pipenv.exceptions.InstallError]:         warnings.warn(
[pipenv.exceptions.InstallError]:       running build
[pipenv.exceptions.InstallError]:       running build_py
[pipenv.exceptions.InstallError]:       creating build
[pipenv.exceptions.InstallError]:       creating build/lib.linux-x86_64-cpython-39
[pipenv.exceptions.InstallError]:       creating build/lib.linux-x86_64-cpython-39/udatetime
[pipenv.exceptions.InstallError]:       copying udatetime/_pure.py -> build/lib.linux-x86_64-cpython-39/udatetime
[pipenv.exceptions.InstallError]:       copying udatetime/__init__.py -> build/lib.linux-x86_64-cpython-39/udatetime
[pipenv.exceptions.InstallError]:       running egg_info
[pipenv.exceptions.InstallError]:       writing udatetime.egg-info/PKG-INFO
[pipenv.exceptions.InstallError]:       writing dependency_links to udatetime.egg-info/dependency_links.txt
[pipenv.exceptions.InstallError]:       writing top-level names to udatetime.egg-info/top_level.txt
[pipenv.exceptions.InstallError]:       reading manifest file 'udatetime.egg-info/SOURCES.txt'
[pipenv.exceptions.InstallError]:       reading manifest template 'MANIFEST.in'
[pipenv.exceptions.InstallError]:       writing manifest file 'udatetime.egg-info/SOURCES.txt'
[pipenv.exceptions.InstallError]:       warning: build_py: byte-compiling is disabled, skipping.
[pipenv.exceptions.InstallError]:       
[pipenv.exceptions.InstallError]:       running build_ext
[pipenv.exceptions.InstallError]:       building 'udatetime.rfc3339' extension
[pipenv.exceptions.InstallError]:       creating build/temp.linux-x86_64-cpython-39
[pipenv.exceptions.InstallError]:       creating build/temp.linux-x86_64-cpython-39/src
[pipenv.exceptions.InstallError]:       x86_64-linux-gnu-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -ffile-prefix-map=/build/python3.9-FZ7wim/python3.9-3.9.5=. -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -D_PYTHON3=1 -I/home/twista/.local/share/virtualenvs/tasks-Ja8_01_a/include -I/usr/include/python3.9 -c ./src/rfc3339.c -o build/temp.linux-x86_64-cpython-39/./src/rfc3339.o -Ofast -std=c99
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c: In function ‘_gettime’:
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c:100:18: error: storage size of ‘t’ isn’t known
[pipenv.exceptions.InstallError]:         100 |     struct timeb t;
[pipenv.exceptions.InstallError]:             |                  ^
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c:101:5: warning: implicit declaration of function ‘ftime’; did you mean ‘ctime’? [-Wimplicit-function-declaration]
[pipenv.exceptions.InstallError]:         101 |     ftime(&t);
[pipenv.exceptions.InstallError]:             |     ^~~~~
[pipenv.exceptions.InstallError]:             |     ctime
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c:100:18: warning: unused variable ‘t’ [-Wunused-variable]
[pipenv.exceptions.InstallError]:         100 |     struct timeb t;
[pipenv.exceptions.InstallError]:             |                  ^
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c: At top level:
[pipenv.exceptions.InstallError]:       ./src/rfc3339.c:516:21: warning: ‘CAPI’ initialized and declared ‘extern’
[pipenv.exceptions.InstallError]:         516 | extern RFC3999_CAPI CAPI = {
[pipenv.exceptions.InstallError]:             |                     ^~~~
[pipenv.exceptions.InstallError]:       error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
[pipenv.exceptions.InstallError]:       [end of output]
[pipenv.exceptions.InstallError]:   
[pipenv.exceptions.InstallError]:   note: This error originates from a subprocess, and is likely not a problem with pip.
[pipenv.exceptions.InstallError]: error: legacy-install-failure
[pipenv.exceptions.InstallError]: 
[pipenv.exceptions.InstallError]: × Encountered error while trying to install package.
[pipenv.exceptions.InstallError]: ╰─> udatetime
[pipenv.exceptions.InstallError]: 
[pipenv.exceptions.InstallError]: note: This is an issue with the package mentioned above, not pip.
[pipenv.exceptions.InstallError]: hint: See above for output from the failure.
ERROR: Couldn't install package: udatetime
```

</details>

--- a/src/rfc3339.c
+++ b/src/rfc3339.c
@@ -10,6 +10,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/timeb.h>
 #include <time.h>
 
 

--- a/src/rfc3339.c
+++ b/src/rfc3339.c
@@ -10,9 +10,11 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/timeb.h>
 #include <time.h>
 
+#ifdef HAVE_FTIME
+#include <sys/timeb.h>
+#endif
 
 #define RFC3339_VERSION "0.0.16"
 #define DAY_IN_SECS 86400

